automatic/wps-office-free/update_helper.ps1

```diff
@@ -17,55 +17,62 @@ return $PackageUrl
 
 }
 
+function Get-ResultInformation() {
+param(
+	[string]$url32,
+	[string]$file,
+	[string]$algorithm = 'sha512',
+	[switch]$info
+)
+
+  $dest = "$env:TEMP\$file"
+  Invoke-WebRequest -UseBasicParsing -Uri $url32 -OutFile $dest
```

Comment:
This will break if the connection requires a proxy.

Reply:
@paulby
Just curious but when has a appveyor session used a proxy?

Reply:
@pauby
Please refer to helper used in `win10mct` package about your query.